src/main/java/uk/gov/hmcts/reform/sscs/reference/data/mappings/InterpreterLanguage.java

```diff
@@ -0,0 +1,320 @@
+    ORM_ORM("orm", "Oromo", null),
+    BFZ_BFZ("bfz", "Pahari", null),
+    PAM_PAM("pam", "Pampangan", null),
+    PAG_PAG("pag", "Pangasinan", null),
+    PAT_PAT("pat", "Patois", null),
+    POL_POL("pol", "Polish", null),
+    POR_POR("por", "Portuguese", null),
+    PHR_PHR("phr", "Pothohari", null),
+    PAN_PAN("pan", "Punjabi", null),
+    PAN_INDIAN("pan-indian", "Punjabi", "Indian"),
+    PAN_PAKISTANI("pan-pakistani", "Punjabi", "Pakistani"),
+    PUS_PUS("pus", "Pushtu (Also Known As Pashto)", null),
+    RMM_RMM("rmm", "Roma", null),
+    RUM_RUM("rum", "Romanian", null),
+    ROM_ROM("rom", "Romany", null),
+    CGG_CGG("cgg", "Rukiga", null),
+    NYO_NYO("nyo", "Runyoro", null),
+    RUS_RUS("rus", "Russian", null),
+    TTJ_TTJ("ttj", "Rutoro", null),
+    SKR_SKR("skr", "Saraiki (Seraiki)", null),
+    KRN_KRN("krn", "Sarpo", null),
+    HBS_HBS("hbs", "Serbo-Croatian", null),
+    TSN_TSN("tsn", "Setswana", null),
+    SCL_SCL("scl", "Shina", null),
+    SNA_SNA("sna", "Shona", null),
+    SND_SND("snd", "Sindhi", null),
+    SIN_SIN("sin", "Sinhalese", null),
+    SLO_SLO("slo", "Slovak", null),
+    SLV_SLV("slv", "Slovenian", null),
+    SOM_SOM("som", "Somali", null),
+    SPA_SPA("spa", "Spanish", null),
+    SUS_SUS("sus", "Susu", null),
+    SWA_SWA("swa", "Swahili", null),
+    SWA_KIBAJUNI("swa-kibajuni", "Swahili", "Kibajuni"),
+    SWA_BRAVANESE("swa-bravanese", "Swahili", "Bravanese"),
+    SWE_SWE("swe", "Swedish", null),
+    SYL_SYL("syl", "Sylheti", null),
+    TGL_TGL("tgl", "Tagalog", null),
+    TAI_TAI("tai", "Taiwanese", null),
+    TAM_TAM("tam", "Tamil", null),
+    TEL_TEL("tel", "Telugu", null),
+    TEM_TEM("tem", "Temne", null),
+    THA_THA("tha", "Thai", null),
+    TIB_TIB("tib", "Tibetan", null),
+    TIG_TIG("tig", "Tigre", null),
+    TIR_TIR("tir", "Tigrinya", null),
+    DON_DON("don", "Toura", null),
+    TUR_TUR("tur", "Turkish", null),
+    TUK_TUK("tuk", "Turkmen", null),
+    TWI_TWI("twi", "Twi", null),
+    UIG_UIG("uig", "Uighur", null),
+    UKR_UKR("ukr", "Ukrainian", null),
+    URD_URD("urd", "Urdu", null),
+    URH_URH("urh", "Urohobo", null),
+    UZB_UZB("uzb", "Uzbek", null),
+    VIE_VIE("vie", "Vietnamese", null),
+    VSA_VSA("vsa", "Visayan", null),
+    WEL_WEL("wel", "Welsh", null),
+    WOL_WOL("wol", "Wolof", null),
+    XHO_XHO("xho", "Xhosa", null),
+    YOR_YOR("yor", "Yoruba", null),
+    ZAG_ZAG("zag", "Zaghawa", null),
+    ZZA_ZZA("zza", "Zaza", null),
+    ZUL_ZUL("zul", "Zulu", null);
+
+    private final String hmcReference;
+    private final String language;
+    private final String dialect;
+
+    public static InterpreterLanguage getLanguageAndConvert(String languages) {
+        return Arrays.stream(InterpreterLanguage.values())
+                .filter(c -> c.getLanguage().equals(languages))
+                .findFirst()
+                .orElse(conditionNotMet(languages));
+    }
+
+    //TODO: Finish of list of lanuages SSCS-10445
```

Comment:
We shouldn't have todo's in PR's

Reply:
We have TODOs in a lot of places as it refers to the ticket it will get done with.  Can remove but was told to add